application/ui/src/routes/inspect/inspect.tsx

```diff
@@ -1,27 +1,39 @@
 // Copyright (C) 2025 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 
+import { useProjectIdentifier } from '@geti-inspect/hooks';
 import { Grid } from '@geti/ui';
 
+import { InferenceProvider } from '../../features/inspect/inference-provider.component';
+import { InferenceResult } from '../../features/inspect/inference-result.component';
+import { SelectedMediaItemProvider } from '../../features/inspect/selected-media-item-provider.component';
 import { Sidebar } from '../../features/inspect/sidebar.component';
-import { StreamContainer } from '../../features/inspect/stream/stream-container';
+/*import { StreamContainer } from '../../features/inspect/stream/stream-container';*/
```

Comment:
Let's remove this import (and the commented code) until we have integrated the full pipeline.